src/ResourceManager/MachineLearningCompute/Commands.MachineLearningCompute/Cmdlets/RemoveAzureRmMlOpCluster.cs

```diff
@@ -58,26 +58,47 @@ public class RemoveAzureRmMlOpCluster : MachineLearningComputeCmdletBase
             HelpMessage = ResourceIdParameterHelpMessage)]
         public string ResourceId { get; set; }
 
+        [Parameter(ParameterSetName = ObjectParameterSet,
```

Comment:
If the parameter is in all of the parameter sets you don't need a parameter attribute for each parameter set, all you need is:
```
[Parameter(Mandatory = false, HelpMessage = IncludeAllResourcesParameterHelpMessage)]
public SwitchParameter IncludeAllResources { get; set; }
````

Reply:
Good to know!